src/platform/user/profile/vet360/containers/VAPProfileField.jsx

```diff
@@ -349,16 +369,8 @@ class VAPProfileField extends React.Component {
             OK
           </button>
         </Modal>
-        <Vet360Transaction
```

Comment:
The fix was to stop wrapping the edit and address validation views in the `Vet360Transaction` component. The `Vet360Transaction` component would re-render when its associated transaction was created, thus remounting the edit view, initializing it with the initial form data which is what caused the form fields to revert.